platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileReaderPbj.java

```diff
@@ -316,10 +318,11 @@ protected void openNewFileChannel(final int index) throws IOException {
      */
     protected void reopenFileChannel(final int index, final FileChannel closedChannel) throws IOException {
         assert index < MAX_FILE_CHANNELS;
-        // May be closedChannel or may be already reopened in a different thread
-        assert fileChannels.get(index) != null;
```

Comment:
Just for discussion, would it be reasonable to have an `if (fileChannels.get(index) == null) return;` type of test instead of the assert to shortcut the file channel open/close combination?

Reply:
Yes, it would be reasonable to have this kind of check because we don't ever pass `null` value in `closedChannel` parameter.